README.md

```diff
@@ -88,7 +88,7 @@ Use the latest possible version of this module that is compatible with the Cyclo
 
 | Version | Schema Version | Format(s) |
 |---------|----------------|-----------|
-| 1.0x | CycloneDX v1.4 | XML/JSON |
+| 1.2.x   | CycloneDX v1.6 | XML/JSON |
```

Comment:
outputting CDX 1.6 instead of 1.5 - i'd consider this a breaking change.
this wold mean, that the version bump should be 2x, not 1.2.x

Reply:
there is no API so I am not clear that there can be any concept of a breaking change. The output to the customer is always a CycloneDX SBOM.

Reply:
> there is no API so I am not clear that there can be any concept of a breaking change. The output to the customer is always a CycloneDX SBOM.

the code provides no API, sure. instead, the code provides a CLI - and this CLI has switches/options, arguments, and behavior. 
examples for breaking changes in the CLI:
- exit codes changes - as if you added new `throw`/`raise` to a function
- altering/removing switches/options - as if you changed a function's/procedure's parameters
- altering the tools behavior - as if you changed how a function/procedure processes
- the result/output changes - as if you changed a function's return 

in this case, you are changing behavior - it no longer writes a file in CycloneDX 1.4, but CycloneDX 1.6